src/esp/physics/bullet/BulletURDFImporter.cpp

```diff
@@ -615,6 +616,12 @@ Mn::Matrix4 BulletURDFImporter::convertURDF2BulletInternal(
 
       world1->addCollisionObject(col, collisionFilterGroup,
                                  collisionFilterMask);
+
+      // TODO: include the articulated object id here
```

Comment:
While this would be nice, I guess it's very inconvenient at this spot in the code. I'll remove this comment before merging.